<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewsTracker</class>
 <widget class="QWidget" name="NewsTracker">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>678</width>
    <height>454</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>678</width>
    <height>454</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>678</width>
    <height>454</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; color:#0000ff;&quot;&gt;정치&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>175</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; color:#0000ff;&quot;&gt;경제&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>305</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; color:#0000ff;&quot;&gt;과학&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>1.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>2.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>3.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>4.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>5.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Date : </string>
   </property>
  </widget>
  <widget class="QCheckBox" name="auto_chk">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>10</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>자동 새로고침</string>
   </property>
  </widget>
  <widget class="QPushButton" name="reload_btn">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>10</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>새로고침</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>2.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>1.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>5.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>3.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>4.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>2.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>1.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>420</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>5.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>3.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>4.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>45</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; color:#0000ff;&quot;&gt;세계&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>100</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>2.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_22">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>160</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>5.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_23">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>80</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>1.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_24">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>140</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>4.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_25">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>120</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>3.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_26">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>175</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; color:#0000ff;&quot;&gt;사회&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_27">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>230</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>2.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_28">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>290</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>5.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_29">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>210</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>1.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_30">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>270</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>4.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_31">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>250</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>3.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_32">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>305</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; color:#0000ff;&quot;&gt;생활/문화&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_33">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>360</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>2.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_34">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>420</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>5.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_35">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>340</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>1.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_36">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>400</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>4.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_37">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>380</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>3.</string>
   </property>
  </widget>
  <widget class="QLabel" name="Date_label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="pol_1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>301</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextBrowserInteraction</set>
   </property>
  </widget>
  <widget class="QLabel" name="pol_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>301</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextBrowserInteraction</set>
   </property>
  </widget>
  <widget class="QLabel" name="pol_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>301</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextBrowserInteraction</set>
   </property>
  </widget>
  <widget class="QLabel" name="pol_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>301</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextBrowserInteraction</set>
   </property>
  </widget>
  <widget class="QLabel" name="pol_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>301</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextBrowserInteraction</set>
   </property>
  </widget>
  <widget class="QLabel" name="eco_1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>301</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextBrowserInteraction</set>
   </property>
  </widget>
  <widget class="QLabel" name="eco_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>301</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextBrowserInteraction</set>
   </property>
  </widget>
  <widget class="QLabel" name="eco_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>250</y>
     <width>301</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextBrowserInteraction</set>
   </property>
  </widget>
  <widget class="QLabel" name="eco_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>270</y>
     <width>301</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextBrowserInteraction</set>
   </property>
  </widget>
  <widget class="QLabel" name="eco_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>290</y>
     <width>301</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextBrowserInteraction</set>
   </property>
  </widget>
  <widget class="QLabel" name="it_1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>340</y>
     <width>301</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextBrowserInteraction</set>
   </property>
  </widget>
  <widget class="QLabel" name="it_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>360</y>
     <width>301</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextBrowserInteraction</set>
   </property>
  </widget>
  <widget class="QLabel" name="it_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>380</y>
     <width>301</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextBrowserInteraction</set>
   </property>
  </widget>
  <widget class="QLabel" name="it_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>400</y>
     <width>301</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextBrowserInteraction</set>
   </property>
  </widget>
  <widget class="QLabel" name="it_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>420</y>
     <width>301</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextBrowserInteraction</set>
   </property>
  </widget>
  <widget class="QLabel" name="glo_1">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>80</y>
     <width>301</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextBrowserInteraction</set>
   </property>
  </widget>
  <widget class="QLabel" name="soc_1">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>210</y>
     <width>301</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextBrowserInteraction</set>
   </property>
  </widget>
  <widget class="QLabel" name="lif_1">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>340</y>
     <width>301</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextBrowserInteraction</set>
   </property>
  </widget>
  <widget class="QLabel" name="glo_2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>100</y>
     <width>301</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextBrowserInteraction</set>
   </property>
  </widget>
  <widget class="QLabel" name="glo_3">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>120</y>
     <width>301</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextBrowserInteraction</set>
   </property>
  </widget>
  <widget class="QLabel" name="glo_4">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>140</y>
     <width>301</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextBrowserInteraction</set>
   </property>
  </widget>
  <widget class="QLabel" name="glo_5">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>160</y>
     <width>301</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextBrowserInteraction</set>
   </property>
  </widget>
  <widget class="QLabel" name="soc_2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>230</y>
     <width>301</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextBrowserInteraction</set>
   </property>
  </widget>
  <widget class="QLabel" name="soc_3">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>250</y>
     <width>301</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextBrowserInteraction</set>
   </property>
  </widget>
  <widget class="QLabel" name="soc_4">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>270</y>
     <width>301</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextBrowserInteraction</set>
   </property>
  </widget>
  <widget class="QLabel" name="soc_5">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>290</y>
     <width>301</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextBrowserInteraction</set>
   </property>
  </widget>
  <widget class="QLabel" name="lif_2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>360</y>
     <width>301</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextBrowserInteraction</set>
   </property>
  </widget>
  <widget class="QLabel" name="lif_3">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>380</y>
     <width>301</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextBrowserInteraction</set>
   </property>
  </widget>
  <widget class="QLabel" name="lif_4">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>400</y>
     <width>301</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextBrowserInteraction</set>
   </property>
  </widget>
  <widget class="QLabel" name="lif_5">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>420</y>
     <width>301</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextBrowserInteraction</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
